public/locales/en/common.json

```diff
@@ -1,3 +1,7 @@
 {
-  "cancel": "Cancel"
```

Comment:
Not sure I understand the common-ness of these. Aren't these just session recording, and the cancel is also in the file menu? Imo it's probably better to not try and generalize too much.

Reply:
This needs some discussion, then. 

Personally, I'm for having strings that are commonly used for certain buttons and components in the `common` file, to make sure that they are consistent across the user interface. It also prevents us/the translator from translate the same word over and over again which could be tedious. But to me, consistency is the main reason. 

Here are some other examples of strings I added to the common.json file as part of PR #144. Let's discuss?
![image](https://github.com/user-attachments/assets/7c602739-77ea-4169-b908-7e9793958f59)

Reply:
After a seconds look a tthese, I agree that many of these are not that common. Play, pause, record, resume should be moved to the session recording panel json